stream_script.py

```diff
@@ -3,17 +3,20 @@
 import webbrowser
 from dotenv import load_dotenv
 import time
+import asyncio
 
 load_dotenv()
 
 
 LIVEKIT_TOKEN_FOR_DIRECTAI = os.getenv("LIVEKIT_TOKEN_FOR_DIRECTAI")
 LIVEKIT_TOKEN_FOR_USER = os.getenv("LIVEKIT_TOKEN_FOR_USER")
+LIVEKIT_TOKEN_FOR_RESULTS = os.getenv("LIVEKIT_TOKEN_FOR_RESULTS", None)
 LIVEKIT_WS_URL = os.getenv("LIVEKIT_WS_URL")
 DIRECTAI_CLIENT_ID = os.getenv("DIRECTAI_CLIENT_ID")
 DIRECTAI_CLIENT_SECRET = os.getenv("DIRECTAI_CLIENT_SECRET")
 
-DIRECTAI_BASE_URL = "https://api.free.directai.io"
+# DIRECTAI_BASE_URL = "https://api.free.directai.io"
+DIRECTAI_BASE_URL = "http://100.66.146.61:8000"
```

Comment:
i'm assuming we don't want this to be public

Reply:
it's a local network URL, not accessible outside our VPN, so should be fine .. although perhaps not best form, maybe this should be taken from the .env file? thought it might be confusing though to say that the URL is an env variable ..

Reply:
i'm fine with hardcoding the base url or setting the url in the env file, just don't want to merge the local IP to `main`

Reply:
oh I forgot that this was in the version I was trying to merge .. my bad .. I'll leave it hardcoded pointing to free for now

Reply:
worth noting that this URL won't work until we've merged the backend changes to prod .. perhaps we should just wait to merge this into main until the listed URL works?

Reply:
yeah I'm fine waiting to merge until the URL works